### PR TITLE
PT-3087: deployment automatization

### PR DESCRIPTION
**Pull request by Vectorfield4**
- PT-3087: Platform auto-update environment
- PK-87: Deployment automatization for SDK

### CODE REVIEW

File: deploy-workflow/workflows/sdk/.github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    - name: Build platform-manager-demo
+      run: |
+        npm run build
+    - name: Build platform-manager-ui docs
+      run: |
+        npm run storybook-build:ui
+    - name: Docker Login
+      uses: azure/docker-login@v1
+      with:
+        login-server: ghcr.io
```
Comment (mvktsk) [inline deploy-workflow/workflows/sdk/.github/workflows/main.yml:72]: Use environment variable for **ghcr.io**

File: deploy-workflow/workflows/sdk/.github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      run: |
+          echo "Project $GITHUB_REPOSITORY successfully build and published to $ARTIFACT_URL_DM and $ARTIFACT_URL_UK"
+
+    - name: Invoke VC image deployment workflow for UI Kit
+      if: ${{ github.ref == 'refs/heads/main' && github.event_name != 'workflow_dispatch' }}
+      uses: benc-uk/workflow-dispatch@v1
+      with:
+        workflow: VC deployment
+        token: ${{ secrets.REPO_TOKEN }}
+        inputs: '{ "artifactUrl": "ghcr.io/${{ github.repository }}/${{ env.UI_KIT_CONTAINER }}:${{ steps.image.outputs.taggedVersion }}", "deployConfigPath":".github/deploy/argoDeployUIKit.json" }'
```
Comment (mvktsk) [inline deploy-workflow/workflows/sdk/.github/workflows/main.yml:94]: Use environment variable for **ghcr.io**

File: deploy-workflow/workflows/sdk/.github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    - name: Invoke VC image deployment workflow for Demo manager
+      if: ${{ github.ref == 'refs/heads/main' && github.event_name != 'workflow_dispatch' }}
+      uses: benc-uk/workflow-dispatch@v1
+      with:
+        workflow: VC deployment
+        token: ${{ secrets.REPO_TOKEN }}
+        inputs: '{ "artifactUrl": "ghcr.io/${{ github.repository }}/${{ env.DEMO_MANAGER_CONTAINER }}:${{ steps.image.outputs.taggedVersion }}", "deployConfigPath":".github/deploy/argoDeployDemoManager.json" }'
```
Comment (mvktsk) [inline deploy-workflow/workflows/sdk/.github/workflows/main.yml:102]: Use environment variable for **ghcr.io**

File: deploy-workflow/workflows/platform/.github/workflows/main.yml [hunk 4]
```diff
@@ -167,3 +168,11 @@ jobs:
...
           docker_hub: ${{ env.PUBLISH_TO_DOCKER }}
           update_latest: ${{ env.UPDATE_LATEST_TAG }}
+
+    - name: Invoke VC image deployment workflow
+      if: ${{ github.ref == 'refs/heads/master' && github.event_name != 'workflow_dispatch' }}
+      uses: benc-uk/workflow-dispatch@v1
+      with:
+        workflow: VC deployment
+        token: ${{ secrets.REPO_TOKEN }}
+        inputs: '{ "artifactUrl": "ghcr.io/${{ github.repository }}/${{ env.IMAGE_NAME }}:${{ steps.image.outputs.taggedVersion }}" }'
```
Comment (mvktsk) [inline deploy-workflow/workflows/platform/.github/workflows/main.yml:178]: Use environment variable for **ghcr.io**

File: deploy-workflow/workflows/sdk/.github/workflows/main.yml (hunk 1 above)
Comment (Vectorfield4) [reply deploy-workflow/workflows/sdk/.github/workflows/main.yml]: Done.

**Pull request merged**